linera-service/src/project.rs

```diff
@@ -230,6 +242,26 @@ impl Project {
         (linera_sdk_dep, linera_sdk_dev_dep)
     }
```

Comment:
I think that we could replace these functions with a call to [`cargo-generate`](https://github.com/cargo-generate/cargo-generate).  It [supports being called as a library](https://github.com/cargo-generate/cargo-generate/blob/main/examples/how-to-use-cargo-gen-as-library/src/main.rs) and handles all the substitution for you — complete with a nice developer flow where it prompts for information that isn't provided.

Reply:
Lifted to an issue https://github.com/linera-io/linera-protocol/issues/2579.